src/java.base/share/classes/java/lang/constant/ClassDesc.java

```diff
@@ -122,7 +128,7 @@ static ClassDesc ofInternalName(String name) {
      * not in the correct format
      */
     static ClassDesc of(String packageName, String className) {
-        ConstantUtils.validateBinaryClassName(requireNonNull(packageName));
+        validateBinaryClassName(requireNonNull(packageName));
```

Comment:
```suggestion
        validateBinaryClassName(packageName);
```
the validate call already null-checks.